### ISSUE TITLE
Client is unable to login when being ran on Linux

### ISSUE THREAD

**Issue by JohnConfidence** [2006rs/singleplayer#6]
When running the client it will start up just fine, but you cannot login. I'm using Arch Linux w/ Kernel 5.2.9 and Oracle JDK 8u22. 

When running the client in the Terminal, I get the following error when trying to login:
```java
java.lang.NullPointerException
        at client.Client.a(Unknown Source)
        at client.Client.a(Unknown Source)
        at client.Client.E(Unknown Source)
        at client.aJ.run(Unknown Source)
        at client.Client.run(Unknown Source)
        at java.lang.Thread.run(Thread.java:748)
Exception in thread "Thread-5" java.lang.NullPointerException
        at client.Client.a(Unknown Source)
        at client.Client.E(Unknown Source)
        at client.aJ.run(Unknown Source)
        at client.Client.run(Unknown Source)
        at java.lang.Thread.run(Thread.java:748)
```

**Comment by Mige5** [comment]
Thats weird, does that happen everytime for u?

Anyway the issue is possibly fixed now. fix available in patch after 1.6.2

**Comment by JohnConfidence** [comment]
Yes, this occurs only when one is logging in. It starts up just fine, just will not login. I'll be happy to test the new version and update this accordingly.

**Comment by JohnConfidence** [comment]
This issue is fixed in 1.6.3
